src/Data/Traversable.purs

```diff
@@ -60,6 +62,39 @@ sequenceDefault :: forall t a m. (Traversable t, Applicative m) =>
 sequenceDefault tma = traverse id tma
 
 
+newtype Id a = Id a
+
+runId :: forall a. Id a -> a
+runId (Id a) = a
```

Comment:
Oh, true! Also it might mean changing other dependencies as if there are things depending on `purescript-identity` for `purescript-foldable-traversable` transitively then that wouldn't work anymore either. Nevermind then!